src/_locales/en/messages.json

```diff
@@ -5,11 +5,30 @@
     "ext_default_title": { "message": "The Great Suspender" },
     "ext_cmd_toggle_tab_suspension_description": { "message": "Suspend/Unsuspend active tab" },
     "ext_cmd_toggle_tab_pause_description": { "message": "Pause/Unpause suspension of active tab" },
-    "ext_cmd_suspend_active_window_description": { "message": "Suspend all other tabs in active window" },
+    "ext_cmd_suspend_selected_tabs_description": { "message": "Suspend selected tabs" },
+    "ext_cmd_unsuspend_selected_tabs_description": { "message": "Unsuspend selected tabs" },
+    "ext_cmd_soft_suspend_active_window_description": { "message": "Suspend all other tabs in active window" },
+    "ext_cmd_force_suspend_active_window_description": { "message": "Force suspend all other tabs in active window" },
     "ext_cmd_unsuspend_active_window_description": { "message": "Unsuspend all tabs in active window" },
-    "ext_cmd_suspend_all_windows_description": { "message": "Force suspend all tabs in all windows" },
+    "ext_cmd_soft_suspend_all_windows_description": { "message": "Suspend all tabs in all windows" },
+    "ext_cmd_force_suspend_all_windows_description": { "message": "Force suspend all tabs in all windows" },
     "ext_cmd_unsuspend_all_windows_description": { "message": "Unsuspend all tabs in all windows" },
 
+    // Contextual menu (Javascript code)
```

Comment:
Needs consistent casing (suggest sentence case)

Reply:
Hmmm yeah, this is an interesting one actually. It's like this to mimic the 'Open Link in New Tab' text in the native chrome right-click menu. And this option should never appear side-to-side with the other options. If you right-click on a link, this is the only option that shows.
See screenshot for an example:
![image](https://user-images.githubusercontent.com/1051763/45534183-57d94880-b7c8-11e8-9e72-d125e77512ac.png)

Reply:
Ah. Welp, whatever. Doesn't matter a great deal.